### PR TITLE
feat(docs): Update docs and images for auto-assignment

### PR DESCRIPTION
**Pull request by leeandher**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [x] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [x] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

Update the documentation around auto-assignment and include a section on the suspect commits page. Images were also updated to reflect current UX.

### CODE REVIEW

File: docs/product/issues/ownership-rules/index.mdx [hunk 1]
```diff
@@ -27,19 +27,12 @@ Types of matches available:
...
-`pattern` matching supports unix-style [glob syntax](<https://en.wikipedia.org/wiki/Glob_(programming)>). For example, add `*` to match anything and `?` to match a single character. _This is not a regex._
-
-`owners`
-
-The owner, or list of owners, identified by either the email of the Sentry user or the name of the team, prefixed with `#`. Thus, the owner may be example@company.com or example1@company.com example2@company.com or `#backend-team`. A space separates each owner in the list of owners.
+ - **`type`** - Can be either `path`, `module`, `url`, or `tags.TAG_NAME`, which matches the specified tag.
+ - **`pattern`** - The pattern you're matching on, which supports unix-style [glob syntax](<https://en.wikipedia.org/wiki/Glob_(programming)>). For example, add `*` to match anything and `?` to match a single character. _This is not a regex._ For example:
+   - `src/javascript/*` for a `path` type
+   - `https://www.example.io/checkout` for a `url` type
+   - `Chrome 81.0.*` for a `tags.browser` type
```
Comment (vivianyentran) [inline docs/product/issues/ownership-rules/index.mdx:34]: Could you add an additional example showing how to use `?` and maybe for a `module` type?

File: docs/product/issues/ownership-rules/index.mdx [hunk 2]
```diff
@@ -27,19 +27,12 @@ Types of matches available:
...
+ - **`owners`** - The owner, or list of owners, identified by either the email of the Sentry user or the name of the team, prefixed with `#`. Thus, the owner may be `example@company.com` or `example1@company.com example2@company.com` or `#backend-team`. A space separates each owner in the list of owners.
```
Comment (vivianyentran) [inline docs/product/issues/ownership-rules/index.mdx:35]:
```suggestion
 - **`owners`** - The owner or a list of owners, separated by a space. Owners can be identified by either the Sentry user's email or the name of the team, prefixed with `#`. For example:
   - `example@company.com`
   - `example1@company.com example2@company.com` 
   - `#backend-team`
```

File: docs/product/issues/ownership-rules/index.mdx [hunk 3]
```diff
@@ -158,17 +151,21 @@ Once ownership rules are configured, they're applied in the following ways:
 
 ### Suggested Assignees
 
-On the **Issue Details** page, you'll see suggested assignees based on ownership rules matching the event you're looking at (by default, the **Issue Details** page shows the latest event). Suggested assignees can also be based on [suspect commits](/product/releases/suspect-commits/). You can assign the _issue_ to a suggested assignee by clicking on the suggestion. An event can have multiple suggested assignees if it matches multiple ownership rules.
+On the **Issue Details** page, you'll see suggested assignees based on ownership rules matching the event, and [suspect commits](/product/releases/suspect-commits/). You can manually assign the issue to a suggested assignee by selecting an option on the dropdown. An event can have multiple suggested assignees if it matches multiple ownership rules.
```
Comment (vivianyentran) [inline docs/product/issues/ownership-rules/index.mdx:154]:
```suggestion
On the **Issue Details** page, you'll see suggested assignees based on ownership rules matching the event, and [suspect commits](/product/releases/suspect-commits/). You can manually assign the issue to a suggested assignee by selecting one from the dropdown. An event can have multiple suggested assignees if it matches multiple ownership rules.
```

File: docs/product/issues/suspect-commits/index.mdx [hunk 4]
```diff
@@ -43,11 +43,11 @@ Suspect commits can be enabled using the following integrations:
 In [sentry.io](https://sentry.io):
 
 1. Go to **Settings > Integrations**.
-1. Click the integration you’d like to use.
-1. Configure the integration if you haven’t already.
-1. Add the repository you'd like to use.
+2. Click the integration you’d like to use.
+3. Configure the integration if you haven’t already.
+4. Add the repository you'd like to use.
```
Comment (vivianyentran) [inline docs/product/issues/suspect-commits/index.mdx:48]: Not a big deal, but I would err on the side of not changing these and keeping them as 1. so it's easier to update and add/remove steps in the future if needed.

File: docs/product/issues/suspect-commits/index.mdx [hunk 5]
```diff
@@ -141,6 +141,27 @@ There are a few reasons why an issue might not have suspect commits:
...
 
-## Limitations
+
+## Where It's Applied
+
+Once suspect commit authors are identified, they're applied in the following ways:
+
+### Suggested Assignees
+
+On the **Issue Details** page, you'll see suggested assignees based on [ownership rules](/product/issues/ownership-rules/) matching the event, and suspect commits. You can manually assign the issue to a suggested assignee by selecting an option on the dropdown.
```
Comment (vivianyentran) [inline docs/product/issues/suspect-commits/index.mdx:151]:
```suggestion
On the **Issue Details** page, you'll see suggested assignees based on [ownership rules](/product/issues/ownership-rules/) matching the event, and suspect commits. You can manually assign the issue to a suggested assignee by selecting one from the dropdown.
```

File: docs/product/issues/suspect-commits/index.mdx [hunk 6]
```diff
@@ -141,6 +141,27 @@ There are a few reasons why an issue might not have suspect commits:
...
+
+### Auto-Assignment
+
+You can automatically assign issues to their suspect commit authors by enabling the "Auto-assign to suspect commits" setting in **[Project] > Settings > Ownership Rules**.
+
+![Select from dropdown to automatically assign issues to their suspect commit authors.](/product/issues/ownership-rules/auto-assign-suspect-commits.png)
+
+If an issue has already been assigned manually, new events will not change the current assignee. If an issue has not been assigned and a suspect commit is found, Sentry assigns it to the commit author.
+
+#### Limitations
```
Comment (vivianyentran) [inline docs/product/issues/suspect-commits/index.mdx:163]:
```suggestion
## Limitations
```
Just so this shows up in the right sidenav

File: docs/product/issues/suspect-commits/index.mdx [hunk 7]
```diff
@@ -141,6 +141,27 @@ There are a few reasons why an issue might not have suspect commits:
...
 
 - If auto-assignment is enabled but the Suspect Committer is not in the Sentry organization, we will not be able to auto-assign the issue to them.
```
Comment (vivianyentran) [inline docs/product/issues/suspect-commits/index.mdx:165]:
```suggestion
- If auto-assignment is enabled but the suspect commit author is not in the Sentry organization, we will not be able to auto-assign the issue to them.
```

**Review by vivianyentran — APPROVED** [review]
some small tweaks, but overall LGTM! Thanks for updating these :)

File: docs/product/issues/suspect-commits/index.mdx (hunk 4 above)
Comment (leeandher) [reply docs/product/issues/suspect-commits/index.mdx:48]: ahh, i had no idea why they were all `1.` but I saw it didn't matter when rendered. TY!

**Pull request merged**